source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -446,13 +446,15 @@ namespace AdaptiveCards
             runtimeclass AdaptiveActionInvoker
             {
                 AdaptiveActionInvoker();
```

Comment:
Do you need to remove this?

Reply:
I'm not certain why? :)
Default constructor is still needed and should work as before:)